.circleci/README.md

```diff
@@ -0,0 +1,69 @@
+# CircleCI 2.0 Workflow Templates
+
+This folder contains template files for CircleCI workflow configurations. You should
+choose the one that you want to use, and name it to be the file `.circleci/config.yml`
+in your repository. Each is described below, along with a description of environment
+variables that you can define in the CircleCI project settings to customize the build.
+
+## Where do I define the variables?
+For each set of variables, some you can define in the config.yml, in the `environment`
+section for a workflow step, for example, here is setting the variable `TIMEZONE`
```

Comment:
For my education: do we have to set a timezone or is it an example of how to set a variable? (and why LA?)

Reply:
You don't have to set a timezone (although I've seen it frequently so it must be useful?) and no reason in particular for LA, if you want to find a different one you like I'm happy to change it :)

Reply:
Maybe we add a comment saying "this isn't strictly necessary"? Just to help people who are trying to take these examples and applying them to their setup. Otherwise you end up with cargo cult kind of things like "Yeah, you gotta run this in the LA timezone because otherwise it gets out of sync with the fizzbuzz-shizzle" :)

Reply:
okay, will add!